observability/logging/log_visualization/logging-kibana.adoc

```diff
@@ -19,7 +19,7 @@ Use and configuration of the Kibana interface is beyond the scope of this docume
 
 [NOTE]
 ====
-The audit logs are not stored in the internal {product-title} Elasticsearch instance by default. To view the audit logs in Kibana, you must use the xref:../../logging/log_storage/logging-config-es-store.adoc#cluster-logging-elasticsearch-audit_logging-config-es-store[Log Forwarding API] to configure a pipeline that uses the `default` output for audit logs.
+The audit logs are not stored in the internal {product-title} Elasticsearch instance by default. To view the audit logs in Kibana, you must use the xref:../../../observability/logging/log_storage/logging-config-es-store.adoc#cluster-logging-elasticsearch-audit_logging-config-es-store[Log Forwarding API] to configure a pipeline that uses the `default` output for audit logs.
```

Comment:
Actually links to "Forwarding audit logs to the log store"

Reply:
Please see my https://github.com/openshift/openshift-docs/pull/74819#discussion_r1577387726.